modules/apps/foundation/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/custom/facet/constants/CustomFacetPortletKeys.java

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
Also, note that the commit message mentions `LSP-80216` but it should be `LPS-80216`.

Reply:
done thanks